contents/blog/aruba-hackathon.md

```diff
@@ -0,0 +1,177 @@
+
+The goal of the team was to create an AI bot that could answer support questions, easing the load on our support heroes and making it easier for the community to find answers to their questions.
+
+Built using Weaviate, Haystack, and `gpt-3.5-turbo`, MaxAI works by taking a user's question, collecting all relevant docs and data for that question, generating a prompt with all that context, and querying OpenAI for an answer.
+
+Here's a flowchart of the process:
+
+```mermaid
+flowchart TD
+    A[User Question] -->|Embed| I(Question Vector)
+    I -->|Query Weaviate|J[Most Similar Docs]
+    J -->|Collect Prompt Params| C{Prompt Context}
+    C --> D[Limitations]
+    C --> E[Personality]
+    C --> F[Context Docs]
+    F --> G[String Prompt]
+    E --> G
+    D --> G
+    G -->|Query OpenAI|H[AI Response]
+```
+
+On the whole, Max gives useful answers – even when using the general data and dealing with complex questions. Max is also dead handy for summarizing long support threads in Slack or GitHub issues. 
+
+![maxai](../images/blog/aruba/max-ai.jpeg)
+
+That said, Max isn't immune to hallucinating solutions – or even URLs for docs that don't exist – if it doesn't know the answer. It's a work in progress we'll be releasing Max on our support Slack soon as a beta. 
+
+Check out the [MaxAI repo](https://github.com/PostHog/max-ai) for more info.
+
+## Dashboard template libraries and learning tracks
+
+- **Team Vibes:** Ian Vanagas, Joe Martin and Andy Vandervell
+
+The marketing team worked together to build (and ship) a [public library of pre-built dashboards](/templates), including an [AARRR pirate metrics dashboard](/templates/aarrr-dashboard), [templates for B2C](/templates/b2c-dashboard) and [B2B products](/templates/b2c-dashboard), and a [landing page report](/templates/landing-dashboard) for marketers transitioning from Google Analytics.
+
+![dashboard templates](../images/blog/aruba/templates.png)
+
+These dashboards are accessible from the 'New Dashboard' modal in PostHog. Some templates require custom events, which you'll be asked to configure before creating the dashboard – you can also change the events you track later.
+
+![dashboard templates](../images/blog/aruba/setup-events.png)
+
+Got a request for a dashboard? DM someone from the marketing on our community Slack.
+
+Joe Martin also worked with Eli from the Website & Docs team to ship [PostHog Tracks](/tracks), a new way to discover tutorials organized by role.
+
+## A built-in data warehouse for PostHog
+
+- **Team DataBeach:** Marius Andra, Frank Hamand and Harry Waye
+
+Dubbed DataBeach because Frank and Harry started building the feature while sipping Piña Coladas by the beach, DataBeach is all about our long-term vision of  [simplifying the modern data stack](/blog/modern-data-stack-sucks) for startups. Less time spent wrangling data = more time shipping products. 
```

Comment:
```suggestion
Dubbed DataBeach because Frank and Harry built it while sipping Piña Coladas by the beach, this is all about our long-term vision of [simplifying the modern data stack](/blog/modern-data-stack-sucks) for startups. Less time spent wrangling data = more time shipping products. 
```